### DISCUSSION TITLE
Bekomme keine Geräte und auch keine Werte im Homeassistant für die Enpal Anlage

### DISCUSSION THREAD

**Discussion by Popperchris** [gickowtf/enpal-homeassistant#17] [Q&A] [answered]
Guten Morgen,
Ich habe vor 2,5 Jahren eine Enpal Photovoltaik Anlage aufbauen lassen. Vor ca 1 Jahr erfolgte ein Besuch eines Technikers der dann auch irgendwas ausgetauscht hat. Mit dem Wechsel der HW hat die IndfluxDB sowie die Grafana Oberfläche die Enpal zur Verfügung stellt eine andere IP.
Jetzt habe ich mich mal mit der Thematik Enpal Werte in den HA einzubinden beschäftigt.
Dafür habe ich Enpal angeschrieben und mir eine "ReadOnly Account" einrichten lassen.
Enpal hat mir daraufhin folgende Infos geschickt:
InfluxDB Server: 192.168.XX.XX
ID: XXXXXXX
User: viewer
Password: XXXXXX
Token:  XXXXX
Rufe ich die URL: 192.168.XX.XX:8086
Sehe ich den login eine InfuxDB 2.0 Version. Melde ich mich mit dem User und Password an komme ich in die Datenbank und sehe alles was mitgeschrieben wird.

das installieren diesen AddOn's funktioniert und ich kann auch die erforderlichen Eingaben machen "Host, Port , Token". Dann sehe ich das Enpal AddOn auch in der Sektion "Konfiguriert". Leider sehe ich aber keinerlei Werte. 
Einen "Debug LogFile" füge ich hier dazu.

Konfiguriere ich als Aden eine InfluDb und erstelle ich da eine entsprechende Konfiguration sehe ich das dass AddOn zu der Datenbank verbunden ist und sehe auch hier Daten. Aber auch hier komme ich irgendwie nicht weiter.

Fehlen mir hier noch irgendwelche Setup Steps oder was ist falsch ?
Wenn noch irgendwelche Infos benötigt werden bitte Bescheid geben das ich die noch zur Verfügung stellen kann.

[home-assistant_enpal_2024-09-04T08-40-42.182Z.log](https://github.com/user-attachments/files/16863520/home-assistant_enpal_2024-09-04T08-40-42.182Z.log)

Vielen Dank für die Hilfe.

Grüße
CL

**Comment by f04ly** [comment]
@gickowtf ich hab genau das gleiche problem. woran könnte das liegen?
![image](https://github.com/user-attachments/assets/0c4515c4-94cc-465f-8ea0-1bfdbf638efd)

**Reply by f04ly** [reply, marked as answer]
ok nevermind... copy und paste hat ein leerzeichen an den token mit kopiert. tip für @Popperchris kopier den token in notepad, dann siehst ob da ein leerzeichen am ende ist und machs weg... dann token wieder in HA und es klappt

**Comment by deverkin** [comment]
Habe dasselbe Problem. 
IP Adresse und Port stimmen, die Integration meldet auch einen Erfolg, aber es werden keine Entitäten hinzugefügt. 
Token habe ich bereits mehrfach neu kopiert. In der Influx DB gibt es ja auch den Button "Copy to Clipboard". Einen Schreibfehler im Token kann ich also ausschließen.
